src/h3lib/lib/coordijk.c

```diff
@@ -253,12 +248,12 @@ void _ijkNormalize(CoordIJK* c) {
  * @return The H3 digit (0-6) corresponding to the ijk unit vector, or
  * INVALID_DIGIT on failure.
  */
-int _unitIjkToDigit(const CoordIJK* ijk) {
+Direction _unitIjkToDigit(const CoordIJK* ijk) {
     CoordIJK c = *ijk;
     _ijkNormalize(&c);
 
-    int digit = INVALID_DIGIT;
-    for (int i = 0; i < 7; i++) {
+    Direction digit = INVALID_DIGIT;
+    for (Direction i = CENTER_DIGIT; i < INVALID_DIGIT; i++) {
```

Comment:
Nit: This makes me think `INVALID_DIGIT` is the wrong name. I'd expect `INVALID_X` to be an opaque value, but here you have to know what the value is (or at least that it's an upper bound). Maybe `OUT_OF_RANGE_DIGIT`? Or define `MAX_DIGIT = IJ_AXES_DIGIT` and use `MAX_DIGIT + 1` here?

Reply:
END_<Something> or LAST_<Something> seem like common patterns in C, I can update to use that instead of (or perhaps as an alias to) INVALID_DIGIT.

Reply:
Makes sense - I think if you use `LAST_DIGIT` you'd need to use `LAST_DIGIT + 1` to indicate the out-of-bounds value, since `LAST_DIGIT` sounds valid.

Reply:
I've actually seen most frequently `NUM_DIRECTIONS` or the equivalent.